aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -257,7 +260,10 @@ Tensor det(const Tensor& self) {
 TORCH_IMPL_FUNC(_linalg_slogdet_out)(const Tensor& A, const Tensor& sign, const Tensor& logabsdet, const Tensor& LU, const Tensor& pivots) {
   // info is an aux tensor
   auto info = at::empty({0}, A.options().dtype(kInt));
-  at::linalg_lu_factor_ex_out(const_cast<Tensor&>(LU), const_cast<Tensor&>(pivots), const_cast<Tensor&>(info), A);
+  // Optimisation: lu_factor_ex requires the input to be F-contig, otherwise it copies
+  // Use the transpose of if A is contiguous since det(A^T) = det(A)
+  // We limit this to real matrices, but it could also be implemented for complex matrices
+  at::linalg_lu_factor_ex_out(const_cast<Tensor&>(LU), const_cast<Tensor&>(pivots), const_cast<Tensor&>(info), A.is_contiguous() && !A.is_complex() ? A.mH() : A);
```

Comment:
Having just `A.mT()` is `A.is_contiguous()` is sufficient, no?

Reply:
Not really as https://github.com/pytorch/pytorch/pull/79743#discussion_r901621302

Reply:
`A.conj().mT()`?